include/flashinfer/activation.cuh

```diff
@@ -30,6 +30,20 @@ __device__ __forceinline__ float silu_kernel(const float& val) {
   return val / (1.0f + __expf(-val));
 }
 
+// https://github.com/NVIDIA/FasterTransformer/blob/d21dc02bc5f70bc7dc0d18ba5801ae263565e68e/src/fastertransformer/kernels/activation_kernels.cu#L37-L57
```

Comment:
Duplicate of

https://github.com/flashinfer-ai/flashinfer/blob/949c32889196a954039a9ce735da07cc70b23a8a/include/flashinfer/math.cuh#L117-L125

Reply:
OK I’ll fix it